packages/client/src/query/aggregations/GroupByMapper.ts

```diff
@@ -27,4 +29,6 @@ export interface GroupByMapper {
   byte: NumericGroupByValue;
   double: NumericGroupByValue;
   integer: NumericGroupByValue;
+  timestamp: TimestampGroupByValue;
+  datetime: DateGroupByValue;
```

Comment:
So dates can technically accept groupby durations smaller than a day, but they give back very weird results (Gateway should probably throw errors on these). For now, just restricting it on our end